rclcpp/include/rclcpp/node_interfaces/node_graph.hpp

```diff
@@ -67,6 +67,10 @@ class NodeGraph : public NodeGraphInterface
   std::map<std::string, std::vector<std::string>>
   get_service_names_and_types() const override;
 
+  RCLCPP_PUBLIC
+  std::map<std::string, std::vector<std::string>>
+  get_action_names_and_types() const override;
```

Comment:
I think this cannot be built since there is no virtual function exists. Could you update `NodeGraphInterface` accordingly?

Reply:
see https://build.ros2.org/job/Rpr__rclcpp__ubuntu_jammy_amd64/887/console